src/reanimated2/NativeMethods.ts

```diff
@@ -1,16 +1,10 @@
-/* global _WORKLET _measure _scrollTo _setGestureState */
+/* global _WORKLET _measure _dispatchCommand _setGestureState */
```

Comment:
is `_scrollTo` no longer present in global?

Reply:
Restored in b13d5c4b172fc9568d822f93e4014a737d3ae617